pkg/wasm/cache_test.go

```diff
@@ -0,0 +1,243 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wasm
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"istio.io/istio/tests/util/leak"
+)
+
+func TestMain(m *testing.M) {
+	leak.CheckMain(m)
+}
+
+func TestWasmCache(t *testing.T) {
```

Comment:
running locally a bunch I see `2540 runs so far, 100 failures (96.21% pass rate). 451.632986ms avg, 1.110633719s max, 292.202541ms min`. Can we just bump the timeout up to like 5s instead of 10ms?

Reply:
Done. thanks!